### PR TITLE
[resolves #2487] Add support for component wordpress

### LINKED ISSUE #2487: Add support for component wordpress

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: itests/standalone/basic/src/test/java/org/wildfly/camel/test/wordpress/subA/WordpressServerHttpRequestHandler.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import org.apache.http.HttpStatus;
+import org.apache.http.client.methods.HttpRequestWrapper;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.protocol.HttpContext;
+import org.apache.http.protocol.HttpRequestHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class WordpressServerHttpRequestHandler implements HttpRequestHandler {
```
Comment (jamesnetherton) [inline itests/standalone/basic/src/test/java/org/wildfly/camel/test/wordpress/subA/WordpressServerHttpRequestHandler.java]: We could maybe just deploy a servlet to handle this. I do this for some of the other components like kubernetes & facebook.

File: feature/extrasA/src/main/resources/modules/system/layers/fuse/com/rometools/rome/1.0/module.xml [hunk 2]
```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module xmlns="urn:jboss:module:1.1" name="com.rometools.rome" slot="1.0">
```
Comment (jamesnetherton) [inline feature/extrasA/src/main/resources/modules/system/layers/fuse/com/rometools/rome/1.0/module.xml]: Is this an intended deletion?

File: itests/standalone/basic/src/test/java/org/wildfly/camel/test/wordpress/subA/WordpressServerHttpRequestHandler.java (hunk 1 above)
Comment (ppalaga) [reply itests/standalone/basic/src/test/java/org/wildfly/camel/test/wordpress/subA/WordpressServerHttpRequestHandler.java]: +1

File: feature/extrasA/src/main/resources/modules/system/layers/fuse/com/rometools/rome/1.0/module.xml (hunk 2 above)
Comment (ppalaga) [reply feature/extrasA/src/main/resources/modules/system/layers/fuse/com/rometools/rome/1.0/module.xml]: I 

> feature/extrasA

rome 1.0 is still there, but not in feature/extrasA. Maybe the removal should have been a part of some previous commit.

**Comment by ppalaga** [comment]
151afc5 uses a servlet instead of mock server.

File: feature/extrasA/src/main/resources/modules/system/layers/fuse/com/rometools/rome/1.0/module.xml (hunk 2 above)
Comment (jamesnetherton) [reply feature/extrasA/src/main/resources/modules/system/layers/fuse/com/rometools/rome/1.0/module.xml]: We can probably get this change in and follow up on that issue later

**Comment by ppalaga** [comment]
70ffcf0 adds a commit to fix the rome 1.0 issue

**Comment by ppalaga** [comment]
3814f4e adds a solution to the rome 1.0 issue

**Comment by jamesnetherton** [comment]
ok to test

**Pull request merged**